crates/sui-framework/packages/bridge/sources/message.move

```diff
@@ -384,6 +388,34 @@ module bridge::message {
         self.op_type
     }
 
+    public fun blocklist_type(self: &Blocklist): u8 {
```

Comment:
I'm conflicted on these names as I understand why they contain the type they're operating over in their names, but also at the same time I wonder if they should be renamed to not include the type name (since it's already part of the function's signature). Don't have super strong opinions here, and naming things is hard...